assets/dev/js/editor/controls/color.js

```diff
@@ -66,57 +46,36 @@ export default class extends ControlBaseDataView {
 		this.$pickerButton = jQuery( this.colorPicker.picker.getRoot().button );
 
 		this.$pickerButton.tipsy( {
-			title: () => this.getControlValue() ? this.getColorObject().displayCode : '',
+			title: () => this.getControlValue() ? this.getNewGlobalData().code : '',
```

Comment:
Don't use `getNewGlobalData` for this purpose

Reply:
Done